.github/workflows/pipeline.yml

```
@@ -160,7 +126,51 @@ jobs:
         pip install versioneer[toml]==0.29
         pip install . --no-deps --no-build-isolation
         conda deactivate
-        python -m unittest discover tests
+        coverage run --omit="conda_subprocess/_version.py,tests/*" -m unittest discover tests
+    - name: cache
+      if: ${{ matrix.python-version }} == 3.12
```

actionlint (1.7.4)

if: condition "${{ matrix.python-version }} == 3.12" is always evaluated to true because extra characters are around ${{ }}

(if-cond)